sympy/printing/glsl.py

```diff
@@ -107,6 +110,8 @@ def _print_MatrixBase(self, mat):
         mat_separator = self._settings['mat_separator']
         mat_transpose = self._settings['mat_transpose']
         glsl_types = self._settings['glsl_types']
+        array_type = self._settings['array_type']
+        array_constructor = Template('%s[${size}]' % array_type)
```

Comment:
`Template` seems like massive overkill here. How about:
```suggestion
        array_constructor = lambda size: '{}[{}]'.format(array_type, size)
```

Reply:
good point :sweat_smile: